utils/plotting_utils.R

```diff
@@ -67,3 +67,31 @@ draw_confusion_matrix <- function(cm, c1_name, c2_name, title="CONFUSION MATRIX"
     text(70, 20, round(as.numeric(cm$overall[2]), 3), cex=1.4)
 
 }  
+
+prettyConfused <- function(Actual, Predict, colors=c("white","red4","dodgerblue3"), text.scl=5){
```

Comment:
Not sure if there are any formatting guidelines for R regarding documentation, but I would add a short description of the function and its arguments.

Reply:
now updated in latest commit: 3fb8701